src/main/java/org/folio/circulation/domain/representations/CheckOutByBarcodeRequest.java

```diff
@@ -6,4 +6,5 @@ private CheckOutByBarcodeRequest() { }
   public static final String ITEM_BARCODE = "itemBarcode";
   public static final String USER_BARCODE = "userBarcode";
   public static final String PROXY_USER_BARCODE = "proxyUserBarcode";
+	public static final String CHECK_OUT_SERVICEPOINT_ID = "checkoutServicePointId";
```

Comment:
This duplicates LoanProperties.CHECKOUT_SERVICE_POINT_ID

Reply:
@julianladisch I think this is an interesting topic, The same property name is used in different contexts, giving us a choice.

Do we want to reduce duplication by reusing an existing constant from the loan context rather than the check out request context, at the risk of confusion or potential accidental change?

Or duplicate the constant definitions so they are defined separately in the different contexts, and can be referenced and evolve separately, at the risk of accidental divergence?

My preference is for the latter.